iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveShell.java

```diff
@@ -216,6 +216,9 @@ private HiveConf initializeConf() {
     // enables vectorization on Tez
     hiveConf.set("tez.mrreader.config.update.properties", "hive.io.file.readcolumn.names,hive.io.file.readcolumn.ids");
 
+    // set lifecycle hooks
+    hiveConf.setVar(HiveConf.ConfVars.HIVE_QUERY_LIFETIME_HOOKS, HiveIcebergCTASHook.class.getName());
```

Comment:
I have seen several hooks already. Maybe we would like to keep them as a single class?

Reply:
Yes, I think @lcspinter has a hook for the miigration rollback. Once that's in, I'll rebase and add my logic to the same hook to be in the same place